### PR TITLE
Revert version tags in README files current released versions

### PR DESCRIPTION
**Pull request by JimSuplizio**

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Shares provide a way to organize sets of files and also can be mounted as an SMB
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-share</artifactId>
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/README.md]: Preview.5 used artifactId azure-storage-file

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file-share/README.md]: Also, it wasn't preview.5 it was preview.4, I think this was double bumped at some point

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file-share/README.md]: Scratch these, I see the other artifact. Approved.

Comment (JimSuplizio) [reply sdk/storage/azure-storage-file-share/README.md]: Just to close out this conversation. 12.0.0-preview.5 is the correct version for [azure-storage-file-share](https://search.maven.org/search?q=a:azure-storage-file-share)

**Pull request merged**